posts/2023-08-30_pharmaverse_story/pharmaverse_story.qmd

```diff
@@ -0,0 +1,103 @@
+I can't disagree with him.
+And I dare say that Ari Siggaard Knoph (Novo Nordisk), Michael Rimler (GSK), Michael Stackhouse (Atorus), Ross Farrugia (Roche), and Sumesh Kalappurakal (Janssen) can't disagree with him either.
+They are the founders of [pharmaverse](https://pharmaverse.org/), members of its Council and kindly shared their memories of how independent companies, in mid-2020, worked together in the creation of a set of packages developed to support the clinical reporting pipeline.
+
+If you are not familiar with this pipeline, the important thing to know is that, in a nutshell, pharmaceutical companies must follow a bunch of standardized procedures and formats (from Clinical Data Interchange Standards Consortium, [CDISC](https://www.cdisc.org/)) when submitting clinical results to Health Authorities.
+Focus on this: **different** companies seeking the **same** standards for outputs.
+
+Paraphrasing Ross *Breaking boundaries through open-source collaboration* presentation in [R/Pharma 2022](https://www.youtube.com/watch?v=APMDOS4v9Hk) and thinking of the development of a new drug, we are talking about a "post-competitive" scenario: the drug has already been discovered and the companies should "just" produce and deliver standardized results.
+
+**Clinical reporting outputs**
+
+Rationally, we can say that companies face the same challenges in these steps of the process.
+Irrationally, we can also say they were working in silos on that before 2018.
+Just as Isaac Newton and Gottfried W. Leibniz [independently developed the theory of infinitesimal calculus](https://en.wikipedia.org/wiki/History_of_calculus), pharmaceutical companies were independently working on R solutions for this pipeline.
+
+But on August 16 and 17 of the mentioned year above, they gathered at the first edition of [R/Pharma conference](https://rinpharma.com/) to discuss R and open-source tooling for drug development (the reasons why open-source is an advantageous approach can be found in this [post](https://pharmaverse.github.io/blog/posts/2023-07-20_cross_company_dev/cross_industry_dev.html) written by Stefan Thoma).
+And according to Isabela Velásquez's article, [Pharmaverse: Packages for clinical reporting workflows](https://posit.co/blog/pharmaverse-packages-for-clinical-reporting-workflows/), one of the most popular questions in this conference was "Is the package code available or on [CRAN](https://cran.r-project.org/)?".
+
+Well, many of them were.
+And not necessarily at that date, but just to mention a few: [*pharmaRTF*](https://atorus-research.github.io/pharmaRTF/index.html) and [*Tplyr*](https://atorus-research.github.io/Tplyr/index.html) from Atorus, [*r2rtf*](https://merck.github.io/r2rtf/) from Merck, [*rtables*](https://insightsengineering.github.io/rtables/main/) from Roche, and etc.
+The thing is that were almost 10000 other packages as well (today, almost 20000).
```

Comment:
This sounds like there are 10 000 packages on CRAN related to clinical reporting.

Reply:
Maybe you could clarify in this sentence that *overall* there were 10 000 packages available on CRAN.